topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,438 @@
+
+To discover these mutations either whole exome sequencing (WES) or whole genome sequencing (WGS) can be used. With these technologies it is possible to uncover the DNA of the parents and offspring to find (shared) mutations in the DNA. These mutations can include insertions/deletions (indels), loss of heterozygosity (LOH), single nucleotide variants (SNVs), copy number variations (CNVs), and fusion genes. 
+
+In this tutorial we will also make use of the HTSGET protocol, which is a program to download our data securely and savely. This protocol has been implemented in the {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} tool, so we don't have to leave galaxy to retrieve our data.
+
+We will not start our analysis from scratch, since the main goal of this tutorial is to use the HTSGET protocol to download variant information from an online archive and to find the causative variant from those variants. If you want to learn how to do the analysis from scratch, using the raw reads, you can have a look at the [Exome sequencing data analysis for diagnosing a genetic disease](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/exome-seq/tutorial.html) tutorial. 
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+> 
+> 1. 
+> {:toc}
+>
+{: .agenda}
+
+
+# Data preperation
+In this tutorial we will use case 5 from the RD-Connect GPAP synthetic datasets. The dataset that we will use consists of WGS VCFs from a real healthy family trio, which originates from the Illumina Platinum initiative {% cite Eberle2017 %} and was made available by the [HapMap project](https://www.genome.gov/10001688/international-hapmap-project). In our dataset a real causative variant was manually spiked-in that should cause breast cancer. The spike-in has been synthetically introduced in the mother and daughter. Here our goal is to identify the genetic variation that is responsible for the disease.
+
+We offer two ways to download the files. Firstly, you can download the files directly from the EGA-archive by requesting DAC access. This will take only 1 workday and gives you access to all of the RD-Connect GPAP synthetic datasets. However if you don't have the time you can also download the data from zenodo.
+
+> ### {% icon comment %} Shortcut: Downloading files from zenodo
+>
+> If you can't wait to get DAC access you can also download the data from zenodo for this tutorial. How to do that is described below.
+>
+> > ### {% icon hands_on %} Hands-on: Retrieve data from zenodo
+> >
+> > 1. Import the 3 VCFs from [Zenodo](https://zenodo.org/record/6483454) to galaxy.
+> >   ```
+> >   https://zenodo.org/record/6483454/files/Case5_F.17.g.vcf.gz
+> >   https://zenodo.org/record/6483454/files/Case5_IC.17.g.vcf.gz
+> >   https://zenodo.org/record/6483454/files/Case5_M.17.g.vcf.gz
+> >   ```
+> >
+> > {% snippet faqs/galaxy/datasets_import_via_link.md %}

```

Comment:
if you indent this to align with the block above, it won't reset the step counter (currently you have two steps labeled as "1" )

Reply:
I noticed, but I was not able to align it with the block above....
Is there a way around this?

Reply:
```suggestion
> >    {% snippet faqs/galaxy/datasets_import_via_link.md %}
```

that should be the correct indentation

Reply:
Thanks! c65eb2ba393676152771dca369b8ca45878d57d2

Reply:
I tried but it seems the snippet will not load properly.
It looks like this: 
`<blockquote class="tip"> <h3 data-toc-skip id="-tip-importing-via-links"> Tip: Importing via links</h3> <ul> <li>Copy the link location</li> <li> <p>Open the Galaxy Upload Manager (galaxy-upload on the top-right of the tool panel)</p> </li> <li>Select Paste/Fetch Data</li> <li> <p>Paste the link into the text field</p> </li> <li> <p>Press Start</p> </li> <li>Close the window</li></ul></blockquote>`
Also, it does not fix the numbering.

Reply:
Apparently, my tabs were 3 spaces big instead of 4 spaces...
Now it's fixed. e58cb8125506cede068681ee931b463c0a731482